web/src/app/shared/services/classifications/classifications.service.ts

```diff
@@ -97,27 +108,15 @@ export class ClassificationsService {
 
   // #endregion
 
-  private classificationParameters(domain: string): QueryParametersModel {
-
-    const parameters = new QueryParametersModel();
-    parameters.SORTBY = TaskanaQueryParameters.parameters.KEY;
-    parameters.SORTDIRECTION = Direction.ASC;
-    parameters.DOMAIN = domain;
-    delete TaskanaQueryParameters.page;
-    delete TaskanaQueryParameters.pageSize;
-
-    return parameters;
-  }
-
   private getClassificationObservable(classificationRef: Observable<any>): Observable<Array<Classification>> {
-    const classificationTypes = this.classificationCategoriesService.getSelectedClassificationType();
+    const classificationTypes: Observable<string> = this.classificationCategoriesService.getSelectedClassificationType();
     return combineLatest(
       [classificationRef,
         classificationTypes]
     ).pipe(
       map(
-        (classification: any, classificationType: any) => {
-          return classification.classifications ? this.buildHierarchy(classification.classifications, classificationType) : [];
+        (classification: any[]) => {
+          return classification[0].classifications ? this.buildHierarchy(classification[0].classifications, classification[1]) : [];
```

Comment:
what if [0] does not exist? At least it looks like a code smell.

Reply:
since we are passing an array with 2 values in the compineLatest() function we always have a length of 2.
Honestly I don't like the change either but I didn't want to invest too much time in looking for alternatives. combineLatest without an array is depreceated. Therefore @BVier switched to an array (which is recommended)

Reply:
Yes it is absolutely a code smell, but this is the best functional hotfix.
In my opinion we should refactor the whole getter method in a seperate ticket.
After this the problem should be resolved.